app/presenters/statistic_presenter.rb

```diff
@@ -24,7 +24,15 @@ def geometry_ratio
   end
 
   def percentage_total_pa_cover
-    percentage_pa_land_cover + percentage_pa_marine_cover
+    perc = ((pa_land_area + pa_marine_area).to_f / ( land_area + marine_area ) * 100).round(2)
```

Comment:
@benkt2 I am just wondering if we would need or benefit from a `rescue nil` - however if the data has good integrity we should never have a divide by zero anyway.